burger-app/src/components/Main/image/index.js

```diff
@@ -0,0 +1,16 @@
+import styled from "styled-components";
+
+const Image = ({ ingredient }) => {
+  return (
+    <ImageStyled
+      src={require(`../../../../assets/images/${ingredient}.png`)}
+      alt="img"
+    />
+  );
+};
```

Comment:
Since you are passing prop `alt` to component, you need to use it instead of hardcoded "img"

```suggestion
const Image = ({ ingredient, alt }) => {
  return (
    <ImageStyled
      src={require(`../../../../assets/images/${ingredient}.png`)}
      alt={alt}
    />
  );
};
```